src/components/routes/Routes.tsx

```diff
@@ -0,0 +1,34 @@
+import { Route, Routes as RouterRoutes } from "react-router-dom";
+import Gift from "@/pages/Gift/Page";
+import Login from "@/pages/Login/Page";
+import NotFound from "@/pages/NotFound/Page";
+import Profile from "@/pages/Profile/Page";
+import AuthRoute from "@/components/routes/AuthRoute";
+import Order from "@/pages/Order/Page";
+
+const Routes = () => {
+  return (
+    <RouterRoutes>
+      <Route path={ROUTE_PATH.HOME} element={<Gift />} />
+      <Route element={<AuthRoute authRequired={false} />}>
+        <Route path={ROUTE_PATH.LOGIN} element={<Login />} />
+      </Route>
```

Comment:
```suggestion
      <AuthRoute required>
        <Route path={ROUTE_PATH.LOGIN} element={<Login />} />
      </Route>
```
이렇게 쓰이는 모습도 상상해보시면 좋을 것 같아요!